Backend/endpoints/referralsFromListingGet.js

```diff
@@ -13,7 +14,46 @@ const {Express} = require('express');
  */
 function referralsGet(app) {
   app.get('/api/referrals/:listingId', async (req, res) => {
-    res.send(await getReferrals(req.params.listingId));
+    if (!req.user?.isManager) {
```

Comment:
I believe this will run into the same issue I had yesterday where `req.user` will be undefined. I fixed it by putting all my objects inside the `req.body` field.